app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -0,0 +1,66 @@
+@import "../tokens/colors";
+$transition: 0.2s cubic-bezier(0.95, 0.05, 0.795, 0.035);
+
+[class^=pb_checkbox_kit] {
+  .pb_checkbox_label {
+    display: block;
+    position: relative;
+    padding-left: 30px;
+    cursor: pointer;
+    font-size: $text_lt_default;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  .pb_checkbox_checkmark {
+    position: absolute;
+    height: 22px;
+    width: 22px;
+    border: solid $border_light 2px;
+    border-radius: 4px;
+    .pb_icon_kit{
```

Comment:
I would suggest adding the prop `classname` to the icon kit, and adding a custom class to adjust the styles for the icon within this kit.

The reason for this suggestion, is because the kit class is dynamically created - if it is updated in the future this kit's scss will break.  If you add a custom class, it will not break.

Example:
```ruby
# In pb_checkbox.rb

pb_icon = Playbook::PbIcon::Icon.new(icon: "check", id: "check_icon", classname: "my_custom_icon", fixed_width: true)
ApplicationController.renderer.render(partial: pb_icon, as: :object)
```

```scss
/* In _checkbox.scss */

.my_custom_icon {
  ...
}
```